index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400|Roboto&display=swap" rel="stylesheet">  
+    <title>Apple- clone</title>
+</head>
+<body>
+    <main class="top container">
+            <nav>
+                <ul>
+                    <li><a href="#"> <img src="assets/Apple-Logo.png" alt="Apple"> </a></li>
+                    <span></span>
+                    <li><a href="#">Store</a></li>
+                    <span></span>
+                    <li><a href="#">Mac</a></li>
+                    <span></span>
+                    <li><a href="#">iPod</a></li>
+                    <span></span>
+                    <li><a href="#">iPhone</a></li>
+                    <span></span>
+                    <li><a href="#">iPad</a></li>
+                    <span></span>
+                    <li><a href="#">iTunes</a></li>
+                    <span></span>
+                    <li><a href="#">Support</a></li>
+                    <span></span>
+                    <li><input type="text"></li>
+                </ul>
+            </nav>
+            <a href="one"><img src="assets/your_verse_title_small.png" alt="What will your verce be?"> </a>
```

Comment:
Use div tag instead img tag. Embed the image on the div tag changing its background.